using_images/s2i_images/dot_net_core.adoc

```diff
@@ -58,6 +63,7 @@ The RHEL 7 images are available through the Red Hat Registry:
 ----
 $ docker pull registry.access.redhat.com/dotnet/dotnetcore-10-rhel7
 $ docker pull registry.access.redhat.com/dotnet/dotnetcore-11-rhel7
+$ docker pull registry.access.redhat.com/dotnet/dotnet-20-rhel7
 ----
```

Comment:
I noticed that the versions were listed from most current to the oldest earlier (2.0, 1.1, and 1.0). Can we reorder this list in the same way for consistency?